src/components/Pagination/index.jsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import ReactPaginate from 'react-paginate';
+
+import classNames from 'classnames';
+import paginationStyles from './Pagination.scss';
+
+import { pageUpdate } from '../../data/actions/assets';
+
+export class Pagination extends React.Component {
+  constructor(props) {
+    super(props);
+    this.onPageClick = this.onPageClick.bind(this);
+  }
+
+  onPageClick(pageData) {
+    this.props.updatePage(pageData.selected);
+  }
+
+  getDisabledScreenReaderText() {
```

Comment:
This is actually not going to be this simple, since the disabled screen reader text will probably be unique for the "..." "button".